www-apps/davical/davical-1.1.7-r1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp
+
+DESCRIPTION="A CalDAV and CardDAV Server"
+HOMEPAGE="https://davical.org/"
+SRC_URI="https://www.davical.org/downloads/${PN}_${PV}.orig.tar.xz -> ${P}.tar.xz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-php/awl-0.59
+	sys-devel/gettext"
+RDEPEND="app-admin/pwgen
+	dev-lang/php:*[calendar,curl,pdo,postgres,xml]
+	dev-perl/DBI
+	dev-perl/DBD-Pg
+	dev-perl/YAML
+	>=dev-php/awl-0.59
+	virtual/httpd-php"
+
+need_httpd
+
+PATCHES=( "${FILESDIR}/${PV}-fix_php4_style_constructors.patch" )
+
+S="${WORKDIR}"
+
+src_compile() {
+	emake built-locale
+}
+
+src_install() {
+	webapp_src_preinst
+
+	dodoc INSTALL CREDITS README TODO ChangeLog
```

Comment:
This could be changed to `einstalldocs`, since it uses default file names: https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html